site/_config.yml

```diff
@@ -126,9 +126,9 @@ host:    127.0.0.1
 sourcerepourl: https://github.com/apache/incubator-edgent
 downloadsurl: /docs/downloads
 projurl: /
-#docsurl: /javadoc/latest
+docsurl: /javadoc/latest
 # above yields things like the following when running on a "jekyll serve" test server
```

Comment:
With your fixes these comments are no longer true so shouldn't they and the orig/wrong commented out docsurl should just be removed?